### ISSUE TITLE
Include channel entities when creating a blueprint

### ISSUE THREAD

**Issue by ceresward** [ceresward/factorio-logiNetChannels#2]
_(Reported on the mod portal:  https://mods.factorio.com/mod/LogiNetChannels/discussion/5d620d6ec54b76000b9cccff )_

> I can't create a blueprint with a network channel assigned, is this normal?

**Comment by 19wolf** [comment]
Any updates on this?

**Comment by ceresward** [comment]
Yes, I posted an update to the [modpage thread](https://mods.factorio.com/mod/LogiNetChannels/discussion/5d620d6ec54b76000b9cccff), I forgot to post it here too:

> Update: I have a new mod, Friend Blueprints, that should help with this. It adds a new blueprinting tool that will select both player and friend force entities. With that tool you can at least create a blueprint that includes the channel entities. Although it doesn't seem to remember the channel setting when the blueprint is stamped, so I may have to tinker with it a little more.
